beeai/agents/triage_agent.py

```diff
@@ -348,24 +344,33 @@ async def check_cve_eligibility(state):
                 available_tools=gateway_tools,
                 issue_key=state.jira_issue
             )
-            state.cve_eligibility_result = json.loads(result)
+            result_data = json.loads(result) if isinstance(result, str) else result
```

Comment:
Isn't the type always known? If not, then maybe some adjustment is needed in `run_tool()`.